src/views/ac-forms/ac-forms-gebruik/components/con-gebruik-step-informatie.js

```diff
@@ -14,17 +73,116 @@ const ConGebruikStepInformatie = ({
   applicatieKeuze,
   selectedApplicatieData,
   setNieuweApplicatieData,
+  isEditMode = false,
+  versionOptions = [],
+  versionsLoading = false,
+  nieuweApplicatie = null,
 }) => {
+  // Get moduleVersie schema for defaults (for new applicatie flow)
+  const moduleVersieSchema = schemas?.moduleversie;
+
+  // Extract default values from schema for new applicatie version
+  const getSchemaDefaults = useCallback(() => {
+    const defaults = {};
+    if (moduleVersieSchema?.properties) {
+      Object.entries(moduleVersieSchema.properties).forEach(([key, property]) => {
+        if (property.default !== undefined) {
+          defaults[key] = property.default;
+        }
+        // Also check for examples as fallback defaults
+        if (property.example !== undefined && defaults[key] === undefined) {
+          defaults[key] = property.example;
+        }
+      });
+    }
+    return defaults;
+  }, [moduleVersieSchema]);
+
+  const schemaDefaults = getSchemaDefaults();
+
+  // Get current version for new applicatie flow
+  const getVersie = useCallback(() => {
+    return Array.isArray(nieuweApplicatie?.moduleVersies) &&
+      nieuweApplicatie.moduleVersies.length > 0
+      ? nieuweApplicatie.moduleVersies[0]
+      : { ...schemaDefaults };
```

Comment:
quick sanity check

is getting the first item of `moduleVersies` proper?
And will this work when editing the gebruik? (in the case that there are more than 1 moduleVersies)